openprescribing/frontend/models.py

```diff
@@ -205,6 +205,7 @@ class Practice(models.Model):
     address5 = models.CharField(max_length=200, null=True, blank=True)
     postcode = models.CharField(max_length=9, null=True, blank=True)
     location = models.PointField(null=True, blank=True)
+    boundary = models.GeometryField(null=True, blank=True, srid=4326)
```

Comment:
For consistency, should we be specifying srid for the precending PointField too?